app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container">
+<h2>Dashboard</h2>
+
+<div class="row">
+  <div class="col-md-4">
+    <h4>Jobs by level of interest</h4>
+    <% @interest.each do |interest, count| %>
+      <li><%= interest %>
+      <span class="badge"><%= count %></span></li>
+    <% end %>
+  </div>
+  <div class="col-md-4">
+    <h4>Top 3 companies by average interest level</h4>
+    <% @average_interest.each do |company, interest| %>
+    <li><%= Company.find(company).name %>
```

Comment:
accessing your models in your views is not best practice. we should query in our controller and pass it in. if that's not do-able here then you could reconsider how you've set your `average_interest` variable up